sw/host/opentitanlib/src/app/config/opentitan.json

```diff
@@ -56,6 +56,25 @@
         }
       ]
     },
+    // This strapping corresponds to `SW_STRAP_RMA_ENTRY` in
```

Comment:
Is it verboten to put a comment in a .json file? I'm not exactly sure where we parse this file, but it seems to work.

Reply:
Aha, it happens in [sw/host/opentitanlib/src/app/config/mod.rs](https://cs.opensource.google/opentitan/opentitan/+/master:sw/host/opentitanlib/src/app/config/mod.rs;l=49;drc=36e476b99baaaa39df34659227b5fde496fa1e34) via `serde_annotate::from_str()`.

The code indicates that `serde_annotate` is very permissive in its JSON parsing. https://github.com/lowRISC/serde-annotate/blob/7e3b69763f57b4cd3e8eef7fa2302bda5a0e6285/src/relax.rs#L17-L19

Reply:
Yeah, JSON doesn't support comments. I know a lot of parsers don't care about them but I think we should remove this - or, alternatively, we can change the suffix to HJSON.